telegram/_inline/inlinequeryresultgif.py

```diff
@@ -47,10 +52,20 @@ class InlineQueryResultGif(InlineQueryResult):
         gif_width (:obj:`int`, optional): Width of the GIF.
         gif_height (:obj:`int`, optional): Height of the GIF.
         gif_duration (:obj:`int`, optional): Duration of the GIF in seconds.
-        thumb_url (:obj:`str`): URL of the static (JPEG or GIF) or animated (MPEG4) thumbnail for
-            the result.
-        thumb_mime_type (:obj:`str`, optional): MIME type of the thumbnail, must be one of
+        thumbnail_url (:obj:`str`): URL of the static (JPEG or GIF) or animated (MPEG4) thumbnail
+            for the result.
+
+            Warning:
+                In Bot API, this is **not** an optional argument. It is formally optional here
+                to allow you to pass the deprecated :paramref:`thumb_url` instead. If you pass
+                neither :paramref:`thumbnail_url` nor :paramref:`thumb_url`, :class:`ValueError`
+                will be raised.
```

Comment:
```suggestion
                The Bot API, does **not** define this as an optional argument. It is formally optional for backwards compatibility with the deprecated :paramref:`thumb_url`. If you pass
                neither :paramref:`thumbnail_url` nor :paramref:`thumb_url`, :class:`ValueError`
                will be raised.
```

similar suggestions eleswhere :)

Reply:
Done